src/app/searchcomponent/searchcomponent.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Output } from '@angular/core'; 
+import {FormControl} from '@angular/forms';
+import {Observable, of , Subscription} from 'rxjs';
+import {debounceTime, map, startWith, switchMap} from 'rxjs/operators';
+ 
+import { HttpClient  } from '@angular/common/http';  
+
+import { environment } from '../../environments/environment';
+import {  EventEmitter } from '@angular/core';
+import { SearchService } from './searchcomponent.service'; 
+ 
+
+@Component({
+  selector: 'app-searchcomponent',
+  templateUrl:'./searchcomponent.component.html',
+  styleUrls: ['./searchcomponent.component.css']
+})
+export class SearchcomponentComponent {
+
+  SUB_SEARCH = environment.mwaNodeServer + '/api/search/food'
```

Comment:
remove environment.mwaNodeServer

Reply:
how to get backend server host?

Reply:
done https://github.com/vnts7/mwa-prj/pull/6/commits/4eb9fdb6141c2aab63f56ec7a7df62b8a66f5ec9